TODO.org

```diff
@@ -31,12 +33,20 @@ Metrics that built around confusion matrix:
 
 - [X] False Omission Rate (FOR)
 
+- [ ] Positive Likelihood 
+
+- [ ] Negative Likelihood
+
 - [X] Prevalence
 
 - [X] F1 Score
 
+- [ ] F Measure (Weighted Harmonic Mean Between Precision And Recall) 
```

Comment:
Same as F1. Ref: https://en.wikipedia.org/wiki/F1_score

Reply:
Should we implement weighted average into F1, as defauly is 0.5?

Reply:
I think we already have mtr_fbeta_score(): https://github.com/chuvanan/metrics/blob/master/R/conf-mat.r#L465.